net.nmlgc.rec98.sh01.yaml

```diff
@@ -0,0 +1,110 @@
+id: net.nmlgc.rec98.sh01
```

Comment:
URL is stuck on rebuilding page

![image](https://github.com/user-attachments/assets/b3c57edc-2dbd-412b-b58b-620e3e635f61)